packages/xstate-test/src/TestModel.ts

```diff
@@ -34,14 +34,6 @@ import {
 } from './utils';
 import { traverseShortestPathsTo } from '@xstate/graph/src/graph';
 
-export interface TestModelDefaults<TState, TEvent extends EventObject> {
-  pathGenerator: PathGenerator<TState, TEvent>;
-}
-
-export const testModelDefaults: TestModelDefaults<any, any> = {
-  pathGenerator: getShortestPaths
```

Comment:
this default became redundant after: https://github.com/statelyai/xstate/pull/3446/files#r967827274